pdf2Video_Converter/Readme.md

```diff
@@ -0,0 +1,36 @@
+# Pdf2Video Converter
+
+## Short description
+
+It will convert take a pdf file and a video as input and Add text audio of pdf to the Given Video.
+
+## Setup instructions
+
+### Required Installation
+
+    pip3 install Pillow
+    pip3 install pytesseract
+    pip3 install pdf2image
+    pip install gTTS
+    pip3 install moviepy
+    pip3 install mutagen   
+```sudo apt-get install tesseract-ocr (for ubuntu)```
```

Comment:
remove these lines (Required installation section) and put all the requirements in the requirements.txt file. Refer to this file for ref: https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Amazon-Price-Alert/requirements.txt